### PR TITLE
[CAM-9090] There are several improvements to the billing page

### PR DESCRIPTION
**Pull request by mirasierra**
those have not been included into the documentation. Please, update the billing page to address for:

- New Pricing section (do not include screenshot
- Update Dashboard to show the new bars by type and filter by type
- Update Usage History section to mention that you will also see MSA and discounts details

@gsanchezu

### CODE REVIEW

File: Cloud Application Manager/Billing/billing-menu.md [hunk 1]
```diff
@@ -34,21 +34,29 @@ All users with Cloud Application Manager organization administrator access.
 * The user should be at the organization level scope to access the Billing option in the left side menu.
 
 ### Billing Dashboard
-When you click on **Billing** in the left side menu you will land into a dashboard page, where a bar graph with the billed amount for the latest months will be shown. You can also change the range of months being shown in the bar graph by selecting a different value from the top-right dropdown on the graph.
+When you click on **Billing** in the left side menu you will land into a dashboard page, where a bar graph with the billed amount for the previous months will be shown. You can change the range of months being shown in the bars graph by selecting a different value in the date filter and you can also select the usage types you want to display with the content type filter..
```
Comment (gsanchezu) [inline Cloud Application Manager/Billing/billing-menu.md]:
```suggestion
When you click on **Billing** in the left side menu you will land into a dashboard page, where a bar graph with the billed amount for the previous months will be shown. You can change the range of months being shown in the bars graph by selecting a different value in the date filter and you can also select the consumption categories you want to display with the consumption category type filter.
```

File: Cloud Application Manager/Billing/billing-menu.md [hunk 2]
```diff
@@ -34,21 +34,29 @@ All users with Cloud Application Manager organization administrator access.
...
 
 ![Billing Dashboard](../../images/cloud-application-manager/billing/billing-dashboard.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Billing/billing-menu.md]: Please, update the screenshot once the labels have been updated (CAM-9106)

File: Cloud Application Manager/Billing/billing-menu.md [hunk 3]
```diff
@@ -34,21 +34,29 @@ All users with Cloud Application Manager organization administrator access.
...
 
 ### Billing Usage History
-When you click on **Usage History** option under the **Billing** left menu item, you will be able to review your detailled billing information.
+When you click on **Usage History** option under the **Billing** left menu item, you will be able to review your detailed billing information.
+When you enter this page, you can see a list with previous months billing summary, with the period of usage and the total credit and debit amounts.
```
Comment (gsanchezu) [inline Cloud Application Manager/Billing/billing-menu.md]:
```suggestion
When you enter this page, you can see a list with the previous months billing summary, including the period of usage and the total credit and debit amounts for each month.
```

File: Cloud Application Manager/Billing/billing-menu.md (hunk 2 above)
Comment (mirasierra) [reply Cloud Application Manager/Billing/billing-menu.md]: I have made it

**Pull request merged**